src/checkers/inference/solver/util/VectorUtils.java

```diff
@@ -0,0 +1,21 @@
+package checkers.inference.solver.util;
+
+import org.sat4j.core.VecInt;
+
+/**
+ * Helper class for creating VecInt and VecInt array.
+ * 
+ * @author jianchu
+ *
+ */
+public class VectorUtils {
```

Comment:
I think this class is specific to MaxSat backend.

Check the usage of this class, if above guess is true, move this class into maxsat package.